### ISSUE TITLE
Add Variation fields

### ISSUE THREAD

**Issue by theferrit32** [clingen-data-model/clinvar-ingest#22]
- [x] name | STRING | NULLABLE
- [x] variation_type | STRING | NULLABLE
- [x] allele_id | STRING | NULLABLE
- [x] release_date | DATE | NULLABLE
- [x] subclass_type | STRING | NULLABLE
- [x] protein_change | STRING | REPEATED
- [x] content | STRING | NULLABLE
- [x] id | STRING | NULLABLE
- [x] descendant_ids | STRING | REPEATED
- [x] num_chromosomes | INTEGER | NULLABLE
- [x] num_copies | INTEGER | NULLABLE
- [x] child_ids | STRING | REPEATED

**Comment by theferrit32** [comment]
@larrybabb did they discontinue the `@NumberOfCopies` field in the XML? (mapped to `num_copies`)

```sql
SELECT count(*) 
FROM `clingen-stage.clinvar_2019_07_31_v1_2_0.variation`
WHERE num_copies is not null
-- 404

SELECT count(*) 
FROM `clingen-stage.clinvar_2019_09_02_v1_1_0_m2.variation`
WHERE num_copies is not null
-- 0

SELECT count(*) 
FROM `clingen-stage.clinvar_2023_11_12_v1_6_61.variation` 
WHERE num_copies is not null
-- 0
```

**Comment by theferrit32** [comment]
Closed in #20
